de-de/bc.html.markdown

```diff
@@ -0,0 +1,101 @@
+---
+language: bc
+contributors:
+    - ["caminsha", "https://github.com/caminsha"]
+filename: learnbc-de.bc
+lang: de-de
+---
+```c
+/* Das is ein mehr-
+zeiliger Kommentar */
+# Das ist ein (einzeiliger) Kommentar (in GNU bc).
+
+    /*1. Variablen und Kontrollstrukturen*/
+num = 45 /* Alle Variablen speichern nur Doubles und es ist
+    nicht möglich String-Konstanten direkt zu speichern */
+num = 45; /* Es kann nach jedem Statement ein optionales Semikolon
+	hinzugefügt werden */
+/* Blöcke werden mit den Operatoren {} (ähnlich wie in C) bezeichnet */
+while(num < 50) {
+	num += 1 /* äquivalent zu num=num+1.
+	a = a Op b ist äquivalent zu a Op= b*/
+}
+/* Ausserdem gibt es ++ (Inkrement) und -- (Dekrement) Operatoren */
+/* Es gibt 3 spezielle Variablen:
+scale: definiert die Anzahl Nachkommastellen
+ibase: definiert die Basis der Eingabe
+obase: definiert die Basis der Ausgabe*/
+/*Wenn-Bedingungen:*/
+hour = read() /*Eingabe einer Zahl*/
+
+if(hour < 12) { /*Operatoren sind genau wie in C*/
+    print "Guten Morgen\n" /*"print" outputs strings or variables
+    print "Guten Morgen\n" /*"print" Gibt Strings oder Variablen
+	mit einem Komma separiert aus.*/
+} else if(hour == 12) {
+    print "Hallo\n"
+	/* Escape-Sequenzen starten mite einem \ in einem String.
+	Um Escape-Sequenzen klarer zu machen, ist hier eine vereinfachte
+	Liste, welche in bc funktioneren.:
+    \b: Backspace
+    \c: Wagenrücklauf (carriage return)
+    \n: Zeilenumbruch
+    \t: Tab
+    \\: Backslash*/
+} else {
+	/* Standardmässig sind Variablen global. */
+    thisIsGlobal = 5
+	/*Variablen können lokal gemacht werden. Benutze das Schlüsselwort "auto" in einer Funktion.*/
+}
+
+/* Jede Variable hat als Voreinstellung den Wert 0. */
+num = blankVariable /*num wurde auf 0 gesetzt.*/
+
+/*Wie in C ist nur 0 falsch.*/
+if(!num) {print "false\n"}
+
+/*Im Gegensatz zu C hat bc die Operatoren ?: nicht. Zum Beispiel
```

Comment:
ternary operator => Ternäroperator?

Reply:
I added the term Ternäroperator, sounds better.